### ISSUE TITLE
table/plot args - enable delayed evaluation

### ISSUE THREAD

**Issue by pawelru** [insightsengineering/teal.widgets#184]
[real-life issue posted on internal R Users chat room]

As an app developer
I order to customise my table title
I want to be able to string concatenate a string literal with objects representing strings in the app environment

Something like: `"this is my custom title with parameter: {{paramcd}}"` into the `teal.modules.clinical::tm_t_tte`.

**Comment by chlebowa** [comment]
I seem to recall something like this already being done in some modules.

**Comment by pawelru** [comment]
> I seem to recall something like this already being done in some modules.

This is true for a default module values ([example](https://github.com/insightsengineering/teal.modules.clinical/blob/main/R/tm_t_tte.R#L154)) evaluated inside module where we have access to the environment. I want to be able to do that from the outside providing appropriate module argument value.

**Comment by chlebowa** [comment]
Oh, I see.
